pkgs/misc/stacki3/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, poetry2nix, fetchFromGitHub }:
+
+poetry2nix.mkPoetryApplication rec {
+  pname = "stacki3";
+  version = "1.0.0";
+
+  projectDir = fetchFromGitHub {
+    owner = "ViliamV";
+    repo = pname;
+    rev = "8b835f7caed5f10b7cf18f4ee50d7b9695f548e8";
```

Comment:
Can you add a comment explaining why we're not using the tag?

Reply:
Can't we fetch from pypi then?

Reply:
Sure, but I will have to introduce `python3Packages` for that.